nursery/connect-to-wmi-namespace-via-wbemlocator.yml

```diff
@@ -0,0 +1,17 @@
+# generated using capa explorer for IDA Pro
+rule:
+  meta:
+    name: connect to WMI namespace via WbemLocator
+    namespace: collection/database/wmi
+    author: michael.hunhoff@fireeye.com
+    scope: function
+    att&ck:
+      - Execution::Windows Management Instrumentation [T1047]
+  features:
+    - and:
+      - basic block:
+        - and:
+          - api: ole32.CoCreateInstance
+          - bytes: 11 F8 90 45 3A 1D D0 11 89 1F 00 AA 00 4B 2E 24 = CLSID_WbemLocator
+          - bytes: 87 A6 12 DC 7F 73 CF 11 88 4D 00 AA 00 4B 2E 24 = IID_IWbemLocator
```

Comment:
guid features!!!

Reply:
should be able to VTGrep these pretty easily

Reply:
![image](https://user-images.githubusercontent.com/156560/117094777-ec01d080-ad21-11eb-9b7f-8877798ba89a.png)


https://www.virustotal.com/gui/search/content%253A%2520%257B87%2520A6%252012%2520DC%25207F%252073%2520CF%252011%252088%25204D%252000%2520AA%252000%25204B%25202E%252024%257D%2520AND%2520size%253A100kb-/files

Reply:
07c57de5f5942433d50d7bf8c6f947c9 `getmac.exe` "Displays NIC MAC information" by MS

Reply:
thank you for checking @williballenthin! `07c57de5f5942433d50d7bf8c6f947c9` splits these calls across multiple functions so the rule doesn't hit for the sample.